### PR TITLE
[AUD-1034] Fix default CoverPhoto tiling

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Fixes default cover photo tiling, by overriding the default DynamicImage `background-size` from `cover` to `auto`. 
Also converts CoverPhoto to ts.

### How Has This Been Tested?

Go to any track with a default cover photo, modify screen size, and confirm it tiles and the resolution stays true.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1034-fix-default-cover-photo-tiling

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1034-fix-default-cover-photo-tiling

File: src/components/general/CoverPhoto.tsx [hunk 1]
```diff
@@ -17,29 +17,49 @@ const messages = {
   imageName: 'Cover Photo'
 }
 
-const CoverPhoto = props => {
+type CoverPhotoProps = {
+  userId: number | null
+  coverPhotoSizes: CoverPhotoSizes | null
```
Comment (piazzatron) [inline src/components/general/CoverPhoto.tsx:22]: lil nit: we have a nullable utility type we've been using for this stuff: `Nullable<number>`

**Review by piazzatron — APPROVED** [review]
LGTM 🚀

**Comment by raymondjacobson** [comment]
I think this PR fixes the immediate issue of not tiling, but I think the resolution is wrong. This picture is probably @2x, so should probably be rendered at 300x300 instead of auto, but that causes some issues with the gradient overlay when it is tiled. Might need revisitation. This PR is a huge improvement & ty for typing in :)

**Comment by dylanjeffers** [comment]
Ahh great callouts, yeah down to revisit sometime, can add a bug in linear for tracking too

**Comment by raymondjacobson** [comment]
Screen Shot 2022-01-04 at 10 59 56 PM
Screen Shot 2022-01-04 at 10 59 41 PM

this is with setting background-size to 300x300 and removing the gradient (and i think w/ the overlay is what we'd want!) but tiling w/ the gradient overlay will be a bit tricky, probably need a separate layer & that will play interestingly with actual images too & the "dynamic image" fading from one to another

**Pull request merged**